### PR TITLE
docs: quality assurance table for return all admin user

### LINKED ISSUE #871: docs: Create Quality Assurance table for 'Returns all admin users' API
## Description

As a developer,
I need to know what are the test cases for a certain API,
so that I can write new unit tests.

As a tester,
I need to know what are the test cases for a certain API,
so that I can perform manual tests on the API.

The task is to come up and add a quality assurance table for "Returns all admin users" API to the docs in the /docs folder.

## Mocks
As done under [docs/quality-assurance-test-cases.md](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md)

mentorship-backend-3

### Update [Required]
- [ ] Add test cases for List All Tasks API.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[2] hours

### PR DESCRIPTION
**Pull request by khushishikhu**
### Description

created quality assurance table for returns all admin user

Fixes #871

### Type of Change:

- Code
- Quality Assurance
- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation
- [x] Update Postman API at /docs folder

### CODE REVIEW

**Comment by khushishikhu** [comment]
@paritoshsinghrahar please review my pr

File: docs/quality-assurance-test-cases.md [hunk 1]
```diff
@@ -202,6 +227,16 @@ Only admin users have access to this.
...
+### Returns all admin users
+
+**Service:** GET /admins
+
+| Test Case        | Outcome |
+| ------------- | ---------------------- |
+| An Admin User with valid access returns all assigned non-admin user which are now admins | Success |
+| A User which is not an admin requests to return all assigned non-admin user | Fail |
+| An Admin User tries to find a non-admin user , when returns all admin users | Fail |
+| An Admin User returns self details | Fail |
```
Comment (paritoshsinghrahar) [inline docs/quality-assurance-test-cases.md]: Please space out the table for consistent text. I know it looks fine when viewed on Github. But to be consistent.

**Review by paritoshsinghrahar — CHANGES_REQUESTED** [review]

File: docs/quality-assurance-test-cases.md (hunk 1 above)
Comment (paritoshsinghrahar) [reply docs/quality-assurance-test-cases.md]: All else looks fine. :smile:

**Review by paritoshsinghrahar — DISMISSED** [review]

**Comment by khushishikhu** [comment]
@paritoshsinghrahar  i have done the requested changes. please have a look now

**Review by paritoshsinghrahar — DISMISSED** [review]
Changes Done. LGTM. Great Effort. 😃

**Review by vj-codes — DISMISSED** [review]
Thank you for your contribution @khushishikhu 
LGTM 🎉

**Comment by vj-codes** [comment]
Adding label ready to merge since it's a documentation change

File: docs/quality-assurance-test-cases.md [hunk 2]
```diff
@@ -174,6 +174,31 @@ This document contains some examples of test cases for each feature implemented
...
+
+**Service:** GET /mentorship_relation/{request_id}/task
+
+| Test Case                                                                                                                                               | Outcome |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| Logged in user requests list all task,or out of all task,for a mentorship relation those which involves the user (as a mentor or as a mentee)           | Success |
+| Logged in user requests list all task,or out of all task, for a mentorship relation those which involves the user (neither as a mentor nor as a mentee) | Fail |
+| Logged in user requests to list all task for a invalid mentorship relation , which involves the user (as a mentor or as a mentee)                       | Fail |     
+| Not logged in user requests to List all task for any valid/invalid mentorship relation                                                                  | Fail |
+
```
Comment (isabelcosta) [inline docs/quality-assurance-test-cases.md]: I think this is not related to this issue right @anitab-org/mentorship-maintainers right?

Comment (vj-codes) [reply docs/quality-assurance-test-cases.md]: @isabelcosta no it's not related, the commits are from a different issue creating tasks table one

Comment (isabelcosta) [reply docs/quality-assurance-test-cases.md]: alright, so i will remove that bit and merge this then @vj-codes :)

File: docs/quality-assurance-test-cases.md [hunk 3]
```diff
@@ -174,6 +174,31 @@ This document contains some examples of test cases for each feature implemented
...
+
```
Comment (isabelcosta) [inline docs/quality-assurance-test-cases.md]:
```suggestion
```

**Review by isabelcosta — APPROVED** [review]
Looks good now!
Thank you so much for contributing @khushishikhu 🙌🏾
